include/dm.h

```diff
@@ -17,79 +23,78 @@
 extern "C" {
 #endif
 
-#include <zephyr/kernel.h>
-#include <stdlib.h>
-#include <zephyr/types.h>
-#include <zephyr/bluetooth/addr.h>
-
 /** @brief Role definition. */
 enum dm_dev_role {
-	/* Undefined role*/
+	/** Undefined role. */
 	DM_ROLE_NONE,
 
-	/* Act as a initiator */
+	/** Act as a initiator. */
 	DM_ROLE_INITIATOR,
 
-	/* Act as a reflector */
+	/** Act as a reflector. */
 	DM_ROLE_REFLECTOR,
 };
 
 /** @brief Ranging mode definition. */
 enum dm_ranging_mode {
-	/* Round trip timing */
+	/** Round trip timing. */
 	DM_RANGING_MODE_RTT,
 
-	/* Multi-carrier phase difference */
+	/** Multi-carrier phase difference. */
 	DM_RANGING_MODE_MCPD,
 };
 
 /** @brief Measurement quality definition. */
 enum dm_quality {
-	/* Good measurement quality */
+	/** Good measurement quality. */
 	DM_QUALITY_OK,
 
-	/* Poor measurement quality */
+	/** Poor measurement quality. */
 	DM_QUALITY_POOR,
 
-	/* Measurement not for use */
+	/** Measurement not for use. */
 	DM_QUALITY_DO_NOT_USE,
 
-	/* Incorrect CRC measurement */
+	/** Incorrect measurement CRC. */
 	DM_QUALITY_CRC_FAIL,
 
-	/* Measurement quality not specified */
+	/** Measurement quality not specified. */
 	DM_QUALITY_NONE,
 };
 
-/** @brief Measurement structure. */
+/** @brief Distance Measurement result structure. */
 struct dm_result {
-	/* Status of the precedure. \a true if measurement was done correctly, \a false otherwise */
+	/** Status of the procedure. \a true if measurement was done correctly,
+	 * \a false otherwise.
+	 */
 	bool status;
 
-	/* Quality indicator */
+	/** Quality indicator */
 	enum dm_quality quality;
 
-	/* Bluetooth LE Device Address */
+	/** Bluetooth LE Device Address. The address is used to distinguish peers. */
```

Comment:
Does Device Address need to be with capital initial letters?

Reply:
I think that Device Address does not need to be with capital initial letters.
I also corrected it in other places.